tests/TranscodingTest.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace IndieHD\AudioManipulator;
+
+use PHPUnit\Framework\TestCase;
+
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+
+use IndieHD\AudioManipulator\Validation\InvalidAudioFileException;
+
+class TranscodingTest extends TestCase
+{
+    /**
+     * @inheritdoc
+     */
+    public function setUp(): void
+    {
+        $this->transcoder = app()->builder->get('transcoder');
+
+        $this->testDir = __DIR__ . DIRECTORY_SEPARATOR . 'samples' . DIRECTORY_SEPARATOR;
+    }
+
+    /**
+     * Ensure that a FLAC file can be transcoded to an MP3 file.
+     *
+     * @return void
+     */
+    public function testTranscodingFlacToMp3Succeeds()
+    {
+        $this->testDir =
```

Comment:
what is this ??

Reply:
An oversight. Oops! It should simply be removed.